.github/workflows/ExportOptions.plist

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>compileBitcode</key>
+	<true/>
+	<key>destination</key>
+	<string>export</string>
+	<key>method</key>
+	<string>ad-hoc</string>
+	<key>provisioningProfiles</key>
+	<dict>
+		<key>vn.luongvo.flutter-survey.staging</key>
```

Comment:
seems like bundle id is different with android: https://github.com/luongvo/flutter-survey/pull/38/files#diff-9526ccfd1d1813ed49c39f8c54dbeb512607376a007d824b905bc8b4e4d202d9R75

Reply:
yes @sleepylee, I had to [update the ios app id to match the app id in distribution cert](vn.luongvo.flutter-survey.staging) on App Dev config. Unfortunately, I can't register a new app id with 5 segments `vn.luongvo.flutter.survey.staging`, then had to go with `vn.luongvo.flutter-survey.staging` 🤪 

This's not an issue, right? 😄

Reply:
not a big deal but in the real scenario, we may have to use `_` instead of `-`; as for android package name we can't use `-`; so this one will create another inconsistency with the android side 🤔

Reply:
I see iOS guys use `-` quite often so that's just an issue on the Android side about package name 🤔 But again, I have to ask Mikey to regenerate app id and necessary certificates again so I think we could go with this 😢